regen/warnings.pl

```diff
@@ -962,6 +973,41 @@ =head2 Default Warnings and Optional Warnings
 Note that neither the B<-w> flag or the C<$^W> can be used to
 disable/enable default warnings.  They are still mandatory in this case.
 
+=head2 "Negative warnings"
+
+As a convenience, you can as of Perl 5.34 pass arguments to the
+C<import()> method both positively and negatively. Negative warnings
+are those with a C<-> sign prepended to their names, positive warnings
+are anything else. This lets you turn on some warnings and turn off
+others in one command. So, assuming that you've already turned on a
+bunch of warnings but want to tweak them a bit in some block, you can
+do this:
+
+    {
+        use warnings qw(uninitialized -redefine);
+        ...
+    }
+
+which is equivalent to:
+
+    {
+        use warnings qw(uninitialized);
+        no warnings qw(redefine);
+        ...
+    }
+
+The argument list is processed in the order you specify. So for example if you
```

Comment:
Grammatical nit: “for example” might be better with commas around it.

(Same applies in warnings.pm.)